misc/tetramesh/src/datastructures/tetrameshbuffers.cpp

```diff
@@ -45,16 +47,17 @@ void TetraMeshBuffers::bind() const {
 
 void TetraMeshBuffers::unbind() const { glBindBuffer(GL_SHADER_STORAGE_BUFFER, 0); }
 
+void TetraMeshBuffers::upload(const TetraMesh& mesh) {
+    const auto& nodeIds = mesh.getNodeIds();
```

Comment:
getNodeIds returns a vector by value right?

Reply:
Guess this upload will only happen once right? wondering if there would be some nice way to avoid the temporary vectors, but I can't really see that at the moment...

Reply:
> getNodeIds returns a vector by value right?

No, only as regular vector. It's constructed on the fly.